api/SVGFEDropShadowElement.json

```diff
@@ -20,7 +20,7 @@
             "version_added": true
           },
           "ie": {
-            "version_added": null
+            "version_added": false
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feDropShadow is all red for IE, so I won't confirm this separately.